src/ReviewPanel.jsx

```diff
@@ -0,0 +1,31 @@
+import React, {Component} from 'react'
+import { Panel, Row, Col } from 'react-bootstrap'
+
+export default class ReviewPanel extends Component {
```

Comment:
Not every component needs to be stateless but there are benefits to using them. I think this is one case where it would be wise to use it.